packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -107,6 +126,60 @@ const GroupedMockCombobox: React.FC<{groupLabelTemplate?: ComboboxProps['groupLa
   );
 };
 
+export const ControlledCombobox: React.FC = () => {
```

Comment:
I noticed this is being exported; did you mean to re-use this with the story?

Reply:
nah, just part of a snippet I use to scaffold a component